components/chart/index.tsx

```diff
@@ -3,19 +3,43 @@ import { CardRecord } from "@/domains/cards/types"
 
 type Props = {
   data: CardRecord[]
+  dataKey1: string
+  dataKey2: string
+  defaultData: { age: number; height_upper: number; height_lower: number }[]
+  defaultDataKey1: string
+  defaultDataKey2: string
```

Comment:
It's not a good idea to pass multiple props using `1`, `2`, ... as suffixes. It's better to use an array for that.
In this case, you can have multiple data passed to the component. In my opinion, the best decision here is to use the following `Props` type

```tsx
type Props<DataRecord extends Record<string, number>> = {
  datas: DataRecord[][]
  xKey: string
  yKey: string
}
```
Notice that we no longer use `CardRecord`, but a generic type that extends `Record<string, number>` (so we can pass any type that is an object with string to number map.) Also, we have an array of arrays which means that we can draw multiple lines on a chart (each outer array is one line).
There is a drawback that you have to transform every data line to one type as `xKey` and `yKey` will be used in each line